ydb/docs/en/core/integrations/toc_i.yaml

```
@@ -1,23 +1,21 @@
 items:
-- name: Log records collection using FluentBit
-  href: fluent-bit.md
-- name: Grafana data source
-  href: grafana.md
-- name: Logstash plugins
-  href: logstash.md
-- name: Schema migrations with goose
-  href: goose.md
-- name: Schema migrations with Liquibase
-  href: liquibase.md
-- name: Schema migrations with Flyway
-  href: flyway.md
-- name: Using Hibernate
-  href: hibernate.md
-- name: Importing from JDBC data sources
-  href: import-jdbc.md
-- name: DBeaver
-  href: ide/dbeaver.md
-- name: DataGrip
-  href: ide/datagrip.md
-- name: Spring Data JDBC
-  href: spring-data-jdbc.md
+- name: Graphical Interface Clients
+  href: gui_clients/index.md
```

1. "GUI" is a well-known abbreviation; let's eliminate the "Clients" and `_clients` parts.
2. It won't be relevant after the previous item, but at the moment, the documentation is in the middle of refactoring from `foo_bar_baz` to `foo-bar-baz` in the URLs, and thus filenames.

```suggestion
- name: Graphical User Interfaces
  href: gui/index.md
```